src/index.html

```diff
@@ -7,6 +7,36 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar">
```

Comment:
```suggestion
    <div class="calendar calendar--30 calendar--tue">
```
Тут должно быть еще два класса-модификатора, которые определяют количество дней в месяце, и день недели первого дня месяца